test-app/tests/integration/components/bs-modal-test.gts

```diff
@@ -380,86 +382,89 @@ module('Integration | Component | bs-modal', function (hooks) {
     },
   );
 
-  module('it animates opening and closing the modal', function () {
-    hooks.before(function () {
-      skipTransition(false);
-    });
+  // TODO #2197: Fix modal animation tests on Safari
+  if (!isSafari) {
+    module('it animates opening and closing the modal', function () {
```

Comment:
Qunit docs state that `modal.if` should be an option, however the @types/qunit packages does not have definitions for this.